.github/workflows/ci.yml

```diff
@@ -97,6 +97,7 @@ jobs:
             cmake --preset default --fresh \
               -DSLANG_SLANG_LLVM_FLAVOR=FETCH_BINARY \
               -DSLANG_SLANG_LLVM_BINARY_URL=$(pwd)/build/dist-release/slang-slang-llvm.zip \
+              -DSLANG_ENABLE_EXAMPLES=OFF \
```

Comment:
I think we should build examples for ci.yml.
And skip it for windows-selfhosted.yml

Can you tell us which CI runs are building examples with your change?

Reply:
The ones building examples now are (seems like all):
ci.yml (new),
compile-regression-test.yml,
falcor-compiler-perf-test.yml,
windows.yml,
linux.yml,
falcor-test.yml,
linux-arm64.yml

Reply:
We should be building the examples only once for each platform.
I am not sure what platforms our CI covers.
Can you figure out which CI setting runs for what platform and OS?
Something like

- ci.yml : Linux_x64_release, and ??
- compile-regression-test.yml : ???
- windows.yml : Windows_x64_release
- linux.yml : ??
- linux-arm64.yml : Linux_arm64_release

I don't think we want to build the examples for falcor-XXX tests.

Reply:
ci.yml:
- Linux GCC x64 **Debug**

compile-regression-test.yml:
- Windows x64 Release

windows.yml:
- Windows x64 Release
- Windows x64 Debug
- Windows Win32 Release
- Windows Win32 Debug

linux.yml:
- Linux GCC x64 Release
- Linux Clang x64 Release
- Linux GCC x64 Debug
- Linux Clang x64 Debug

linux-arm.yml:
- Linux GCC aarch64 Release

Reply:
Yea... it looks like we should build the example for all "windows.yml", "linux.yml" and "linux-arm.yml".
The rest of CI should skip because it will be redundant.